### PR TITLE
Local storage tests page

### PR DESCRIPTION
**Pull request by bwaresiak**
Related task: https://app.asana.com/0/856498667320406/1201941114393612

This test page allows for easy testing of local storage and cookies capabilities.

### CODE REVIEW

File: features/local-storage.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
+
```
Comment (kdzwinel) [inline features/local-storage.html:1]: let's drop the empty line

File: features/local-storage.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      const initialDiv = document.querySelector('#initial');
+      const testsDiv = document.querySelector('#tests');
+
+      startButton.addEventListener('click', () => startIncrementing());
+      cleanupButton.addEventListener('click', () => cleanup());
+
+      initialDiv.innerHTML = countInfoAsHtml();
+
+      function startIncrementing () {
+          setInterval(() => {
```
Comment (kdzwinel) [inline features/local-storage.html:36]: if you'd like to stop this on "cleanup" button click:

```js
let someVariable = setInterval(…)

…

clearInterval(someVariable)
```

File: features/local-storage.html [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Local Storage & Cookies storing</title>
+</head>
+<body>
+    <h1>Local Storage & Cookies storing</h1>
```
Comment (kdzwinel) [inline features/local-storage.html:11]: 1. can we add a description of what this page does / what it is testing exactly?
2. and a button to go back to the index like other pages have

**Review by kdzwinel — APPROVED** [review]
I found couple of nits, feel free to merge after you address them 🤘

File: features/local-storage.html (hunk 2 above)
Comment (bwaresiak) [reply features/local-storage.html:36]: I don't need to, but thanks for the tip :)

**Pull request merged**